db/db.go

```diff
@@ -85,7 +85,7 @@ func (d *DB) BeginBatch() *Batch {
 
 // LiveQuery returns a helper to fire a liveQuery request
 func (d *DB) LiveQuery(col string) *realtime.LiveQuery {
-	return d.realTime.LiveQuery(d.db, col)
+	return realtime.Init(d.config.Project, websocket.Init(d.config.URL, d.config)).LiveQuery(d.db, col)
```

Comment:
This is not right. This will cause a new websocket connection for each livequery fired. Let this be as is